creator-cards/fahad-ashiq.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "My name is Fahad Ashiq!",
+  "type": "I am Software Development Engineer having 3 years of experience starting from system analysis and design ending by implementation, testing, deployment & technical documentation. Have the passion to work and gain experiences, quick learner, having good communication skills, take responsibilities with conscience, flexible and motivated team member",
```

Comment:
That's not what the type is for. Please check against the new format.

Reply:
it's done can you please check it to give it a final review, many thanks!